activerecord/test/cases/adapters/postgresql/postgresql_adapter_test.rb

```diff
@@ -455,6 +455,21 @@ def test_unparsed_defaults_are_at_least_set_when_saving
         end
       end
 
+      def test_only_check_for_insensitive_comparison_capability_once
+        with_example_table "id SERIAL PRIMARY KEY, number INTEGER" do
+          number_klass = Class.new(ActiveRecord::Base) do
+            self.table_name = "ex"
+          end
+          column = number_klass.columns_hash["number"]
+          assert_queries 1, ignore_none: true do
+            @connection.__send__(:can_perform_case_insensitive_comparison_for?, column)
```

Comment:
This can be done by calling a public `case_insensitive_comparison(attribute, value)` instead. Is there an easy way to set up an `attribute`?
I'm on the fence with this because the problem is specifically local to `can_perform_case_insensitive_comparison_for?` in this class.

Reply:
Yeah, would be better not to reach to a private method.